proto/pulumi/language.proto

```diff
@@ -133,6 +133,7 @@ message RunRequest {
     string organization = 12;       // the organization of the stack being deployed into.
     google.protobuf.Struct configPropertyMap = 13; // the configuration variables to apply before running.
     ProgramInfo info = 14; // the program info to use to execute the program.
+    bool debug = 15;                // true if we're running in debug mode.
```

Comment:
Feels like this should be more explicit like `start_debugger`

Reply:
Done :heavy_check_mark: